File: php/utility/MakeContext.php
<?php
declare(strict_types=1);

// DungeonsAndDragons SDK utility: make_context

require_once __DIR__ . '/../core/Context.php';

class DungeonsAndDragonsMakeContext
{
    public static function call(array $ctxmap, ?DungeonsAndDragonsContext $basectx): DungeonsAndDragonsContext
    {
        return new DungeonsAndDragonsContext($ctxmap, $basectx);
    }
}

File: .sdk/tm/rb/utility/make_context.rb
# DungeonsAndDragons SDK utility: make_context
require_relative '../core/context'
module DungeonsAndDragonsUtilities
  MakeContext = ->(ctxmap, basectx) {
    DungeonsAndDragonsContext.new(ctxmap, basectx)
  }
end
